app/models/batch_process.rb

```diff
@@ -47,6 +51,32 @@ def oids
     @oids ||= [oid]
   end
 
+  def output_csv
```

Comment:
thoughts on making this (and created_file_name) a concern? this would mostly be the idea that CsvExportable is a set of features that can be kept together outside the primary model and not necessarily for reuse at this moment, so I'd consider it a tidying thing and not a structural one and definitely deep in the realm of opinion.